System/Random.hs

```diff
@@ -222,11 +244,46 @@ class Monad m => MonadRandom g m where
   uniformWord32 = uniformWord32R maxBound
   uniformWord64 :: g -> m Word64
   uniformWord64 = uniformWord64R maxBound
+  uniformBytes :: Int -> g -> m ByteArray
+  default uniformBytes :: PrimMonad m => Int -> g -> m ByteArray
+  uniformBytes = uniformByteArrayPrim
+  {-# INLINE uniformBytes #-}
```

Comment:
Great. I'm in favour of (1) including `uniformBytes` in the class; (2) having a default implementation in terms of the largest single-item producer (`uniformWord64`); (3) this implementation. Basically, :+1: